OSDOCS-4627: Adds notes for 4.11.18 (#53427)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

[id="ocp-4-11-17-updating"]
==== Updating

To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.

[id="ocp-4-11-18"]
=== RHBA-2022:8698 - {product-title} 4.11.18 bug fix update

Issued: 2022-12-05

{product-title} release 4.11.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2022:8698[RHBA-2022:8698] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:8697[RHBA-2022:8697] advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.11.18 --pullspecs
----

[id="ocp-4-11-18-notable-technical-changes"]
==== Notable Technical Changes

> * Previously named heterogeneous clusters will now be refered to as multi-architecture in {product-title} documentation. For more information, see xref:../post_installation_configuration/deploy-heterogeneous-configuration.adoc[Configuring a multi-architecture cluster]

Reviewer: ```suggestion
* Previously named heterogeneous clusters are now referred to as multi-architecture in {product-title} documentation. For more information, see xref:../post_installation_configuration/deploy-heterogeneous-configuration.adoc[Configuring a multi-architecture cluster]
```

[id="ocp-4-11-18-bug-fixes"]
==== Bug fixes

> * Previously, some object storage instances respond with `204 No Content` when there is no content to display. The {rh-openstack-first} SDK used in {product-title} does not handle 204s correctly. With this update, the installation program works around the issue when there are zero items to list. (link:https://issues.redhat.com/browse/OCPBUGS-4081[*OCPBUGS-4081*])

Reviewer: ```suggestion
* Previously, some object storage instances responded with `204 No Content` when there was no content to display. The {rh-openstack-first} SDK used in {product-title} did not handle 204s correctly. With this update, the installation program works around the issue when there are zero items to list. (link:https://issues.redhat.com/browse/OCPBUGS-4081[*OCPBUGS-4081*])
```

> * Previously, the rollout times for `kube-apiserver` on a loaded cluster were slow and sometimes exceeded the 5 min rollout timeout.

Reviewer: Looks good, I would adjust the hard line wraps in 3034 - 3035.
